packages/orchestrator-ui-components/src/pages/metadata/WFOWorkflowsPage.tsx

```diff
@@ -0,0 +1,252 @@
+import React from 'react';
+import { useTranslations } from 'next-intl';
+import type { Pagination } from '@elastic/eui/src/components';
+
+import {
+    DEFAULT_PAGE_SIZE,
+    DEFAULT_PAGE_SIZES,
+    METADATA_WORKFLOWS_TABLE_LOCAL_STORAGE_KEY,
+    WFOProductBlockBadge,
+} from '../../components';
+import { WFOTableWithFilter } from '../../components';
+import {
+    getTableConfigFromLocalStorage,
+    getDataSortHandler,
+    getPageChangeHandler,
+    getEsQueryStringHandler,
+} from '../../components';
+import type { WFOTableColumns, WFODataSorting } from '../../components';
+
+import type { WorkflowDefinition } from '../../types';
+import { SortOrder } from '../../types';
+
+import { useDataDisplayParams } from '../../hooks';
+import { WFOMetadataPageLayout } from './WFOMetadataPageLayout';
+import {useQueryClient} from "react-query";
+
+const WORKFLOW_FIELD_NAME: keyof WorkflowDefinition = 'name';
+const WORKFLOW_FIELD_DESCRIPTION: keyof WorkflowDefinition = 'description';
+const WORKFLOW_FIELD_TARGET: keyof WorkflowDefinition = 'target';
+const WORKFLOW_FIELD_PRODUCT_TAGS: keyof WorkflowDefinition = 'productTags';
+const WORKFLOW_FIELD_CREATED_AT: keyof WorkflowDefinition = 'createdAt';
+
+export const WFOWorkflowsPage = () => {
+
+    //TODO: Delete when backend is implemented
+
+    const data = {
```

Comment:
Since you are using static test data here, the compiler tries to prerender the page with data and fails (see compilation error in pipeline). For a temporary fix wrap the <WFOMetadataPageLayout> with a <NoSSR> component.

Reply:
Ok, so that was giving me the error. Thanks !